README.md

```diff
@@ -1,106 +1,122 @@
-# Wanderlust - Your Ultimate Travel Blog 🌍✈️
-
-WanderLust is a simple MERN travel blog website ✈ This project is aimed to help people to contribute in open source, upskill in react and also master git.
+<div>
+  <h1>Wanderlust</h1>
+  <h2>The Ultimate Travel Blog 🌍✈ for You </h2>
+</div>
 
 ![Preview Image](https://github.com/krishnaacharyaa/wanderlust/assets/116620586/17ba9da6-225f-481d-87c0-5d5a010a9538)
 
-## [Figma Design File](https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=0%3A1&mode=design&t=c4oCG8N1Fjf7pxTt-1)
-## [Discord Channel](https://discord.gg/FEKasAdCrG)
+<hr>
 
-## 🎯 Goal of this project
+<div>
+<h3>• Watch our collaboration video : </h3>
+  <a href="https://youtu.be/ANfC1u_N_A0?feature=shared"> <img src="https://github.com/krishnaacharyaa/wanderlust/assets/133582566/47d71dd6-0390-479e-9d4e-3f077ef1a987.png" width="45" height="40"></a> <h4> with TechwithShubham </h4> 
+  For an in-depth look! 🎬 Dive into the details and discover more about our exciting project.
+</div>
+<hr>
 
-At its core, this project embodies two important aims:
+## About :
 
-1. **Start Your Open Source Journey**: It's aimed to kickstart your open-source journey. Here, you'll learn the basics of Git and get a solid grip on the MERN stack and I strongly believe that learning and building should go hand in hand.
-2. **React Mastery**: Once you've got the basics down, a whole new adventure begins of mastering React. This project covers everything, from simple form validation to advanced performance enhancements. And I've planned much more cool stuff to add in the near future if the project hits more number of contributors.
+**• WanderLust** is a straightforward MERN travel blog website ✈. This project aims to help people contribute to open source, enhance their  React skills, and master Git. 
+<hr>
 
-_I'd love for you to make the most of this project - it's all about learning, helping, and growing in the open-source world._
+<div>
+<h3>• Figma Links of Our Project :</h3>
+  <a href="https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=0%3A1&mode=design&t=c4oCG8N1Fjf7pxTt-1">
```

Comment:
Remove * and then make this more interesting somehow. 

We need a section like important links which will have yotuube, discord and figma link. Soon after the first image. this should be there.

You can come up with innovation.
I see it something like table with 2 cols, 1st column logo and 2nd column description and link
Discord Logo | Join our project's discord link [here]()
Figma Logo | ---
youtube logo | --

Think about it it should be attractive at the same time appealing

Reply:
okk done